Added note to clarify globbing support

Nyami opened the pull request:
## Summary

#6293 confirms this behaviour so thought it valuable to add a clear note.

Nyami committed:
Added note to clarify globbing support

#6293 confirms this behaviour so thought it valuable to add a clear note

rpetrusha reviewed (changes requested):
Thanks, @Nyami, for adding the note. I've proposed some changes for you to consider.

pkulikov commented on docs/core/tools/dotnet-sln.md:
```diff
@@ -89,3 +89,6 @@ Add multiple C# projects to a solution using a globbing pattern:
 `dotnet sln todo.sln remove **/*.csproj`
+
+> [!NOTE]
+> Globbing is not a CLI feature but rather a feature of the command shell, to sucessfully expand the files you must be using a shell that supports.
```
one more suggestion: 
- "you must be using" --> "use" or "you must use"

Nyami committed:
Added note to clarify globbing support

#6293 confirms this behaviour so thought it valuable to add a clear note

Thraka reviewed (commented):
Would it be helpful to add a `For more information about globbing, see [Wikipedia](https://en.wikipedia.org/wiki/Glob_%28programming%29).` at the end of the note?

rpetrusha commented:
This looks good, @Nyami. Though I like the suggestion from @Thraka that we add a link to more information about globbing. Would you like to do that?

Nyami commented:
@rpetrusha I'm happy to add @Thraka suggestion to the note.

Nyami committed:
Added note clarifying blobbing support

Added note to make it clear that globbing in not a CLI feature along with link to Wikipedia for more info on blobbing

rpetrusha commented:
Thanks, @Nyami, for the contribution and the additional changes. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.
